airbyte-webapp/src/views/Connection/ConnectionForm/components/TransformationField.tsx

```diff
@@ -12,8 +12,9 @@ const TransformationField: React.FC<
   ArrayHelpers & {
     form: FormikProps<{ transformations: Transformation[] }>;
     defaultTransformation: Transformation;
+    mode?: "readonly" | "edit" | "create";
```

Comment:
I see this frequently. It may be worth putting into a type and exporting for reuse:

```
export type ConnectionViewMode = "readonly" | "edit" | "create";
```